### PR TITLE
Feature/vih 5427 reconnect to eventhub

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

[VIH-5427](https://tools.hmcts.net/jira/browse/VIH-5427)

### Change description ###

* check if video needs to be shown/hidden after eventhub reconnect

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/services/api/video-web.service.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ export class VideoWebService {
     }
 
     getConferenceById(conferenceId: string): Observable<ConferenceResponse> {
+        console.trace('boop de bop');
```
Comment (RuwanikaWeerakkody) [inline VideoWeb/VideoWeb/ClientApp/src/app/services/api/video-web.service.ts]: What is this Shaed :)

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/beta-banner/beta-banner.component.ts [hunk 2]
```diff
@@ -1,56 +1,61 @@
...
+    }
+    ngOnInit(): void {
+        this.getInPageFeedbackUrl();
+        this.setupSubscribers();
+    }
 
-  updateFeedbackUrl(feedbackUrl: string): void {
-    this.pageUrl = feedbackUrl + this.router.url;
-  }
+    ngOnDestroy(): void {
```
Comment (RuwanikaWeerakkody) [inline VideoWeb/VideoWeb/ClientApp/src/app/shared/beta-banner/beta-banner.component.ts]: Why are we doing this in beta banner?

File: VideoWeb/VideoWeb/ClientApp/src/app/services/api/video-web.service.ts (hunk 1 above)
Comment (shaed-parkar) [reply VideoWeb/VideoWeb/ClientApp/src/app/services/api/video-web.service.ts]: haha i needed to figure out which test was calling the API rather thana stub

Comment (shaed-parkar) [reply VideoWeb/VideoWeb/ClientApp/src/app/services/api/video-web.service.ts]: removed :)

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/beta-banner/beta-banner.component.ts (hunk 2 above)
Comment (shaed-parkar) [reply VideoWeb/VideoWeb/ClientApp/src/app/shared/beta-banner/beta-banner.component.ts]: we need to manage our subscriptions. I noticed this when i was digging through tests calling the API. If we don't unsubscribe between components then we will get memory leaks. We should either convert these api calls to promises or unsubscribe when the component is destroyed

**Pull request merged**